"0","data_clean |>
  mutate(across(-c(1,2)), as.numeric())
"
"1","[1m[33mError[39m in `mutate()`:[22m
[38;5;255m[33m![38;5;255m Can't transform a data frame with duplicate names.[39m
[90mRun `]8;;x-r-run:rlang::last_trace()rlang::last_trace()]8;;` to see where the error occurred.[39m
"
